.travis.yml

```diff
@@ -34,12 +34,10 @@ rvm:
 services:
   - redis-server
 
-bundler_args: --without development production --retry=3 --jobs=16
-
 install:
   - nvm install
   - npm install -g yarn
-  - bundle install
+  - bundle install --path=vendor/bundle --without development production --retry=3 --jobs=16
```

Comment:
No. Not happening. The original documentation expect you to use the default install script. You may check the Cache in travis. It is empty.